scss/5-pages/_report.scss

```diff
@@ -0,0 +1,80 @@
+
+.site-header {
+	color:#4d4d4d;
+	margin-top: 5rem;
+	margin-bottom: 5rem;
+
+    h1 {
+        font-size: 60pt;
+        margin-top: 60pt;
+        font-weight: bold;
+    }
+
+    h2{
+    	margin-top:0px;
+    }
+
+    margin-bottom: 2em;
+}
+
+
+.container.report {
+	color:#4d4d4d;
+	margin-bottom: 45pt;
```

Comment:
Exactly.
Which is why it should be in px or even better in rem.
Pt makes only sense for print, and since the Photoshop document is 72dpi, we can simply switch pt for px. And apply the rem mixin on it.
Like this:
~~~
@include rem(margin-bottom, 45px);
~~~